app/src/main/java/com/github/factotum_sdp/factotum/ui/login/LoginFormState.kt

```diff
@@ -0,0 +1,10 @@
+package com.github.factotum_sdp.factotum.ui.login
+
+/**
+ * Data validation state of the login form.
```

Comment:
If you could reformulate your explanation for this class it would be great, I have some trouble understanding what it is going to be used for simply from it's description/code.